source/classes.tex

```diff
@@ -6035,9 +6035,10 @@
 \indextext{destruction!member access}%
 For an object with a non-trivial constructor, referring to any non-static member
 or base class of the object before the constructor begins execution results in
-undefined behavior\ubdef{class.cdtor.before.ctor.after.dtor}. For an object with a non-trivial destructor, referring to
+undefined behavior\ubdef{class.cdtor.before.ctor.before.ctor}.
```

Comment:
{class.cdtor.before.ctor.before.ctor} looks like a weird repetition. Did you mean just {class.cdtor.before.ctor} ?

Reply:
Resolved